playbooks/common/openshift-cluster/redeploy-certificates/etcd.yml

```diff
@@ -50,6 +52,7 @@
       etcd_peers: "{{ groups.oo_etcd_to_config | default([], true) }}"
       etcd_certificates_etcd_hosts: "{{ groups.oo_etcd_to_config | default([], true) }}"
       openshift_ca_host: "{{ groups.oo_first_master.0 }}"
+      r_etcd_common_etcd_runtime: "{{ openshift.common.etcd_runtime }}"
```

Comment:
The more we talk about the more it should be `r_etcd_upgrade_etcd_runtime` which sets its `r_etcd_common_etcd_runtime` equivalent in `etcd_upgrade` role. This is a first step before I finish refactoring all etcd like roles. I will take care of it.